### PR TITLE
Added tags component

### PR DESCRIPTION
**Pull request by yuliiashpak**
-added tags component 
-fixed fontSize of error in inputs

### CODE REVIEW

File: src/components/Tags/Tag/index.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+export interface ITagProps extends TouchableOpacityProps {
+  children: React.ReactNode;
+  disabled?: boolean;
+  theme?: any;
+  color?: string;
+  shape?: 'rectangle' | 'rounded' | 'round';
+  size?: 'small' | 'large';
+  withCross?: boolean; 
+  shadow?: boolean;
+  props?: any;
```
Comment (zaporozhetsAnton) [inline src/components/Tags/Tag/index.tsx:18]: why do we need this line? `props?: any;`

File: src/components/Tags/index.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+export interface ITagsProps {
+  tags: ITagData[];
+  theme?: any;
+  color?: string;
+  shape?: 'rectangle' | 'rounded' | 'round';
+  size?: 'large' | 'small';
+  withCross?: boolean; 
+  shadow?: boolean;
+  props?: any;
```
Comment (zaporozhetsAnton) [inline src/components/Tags/index.tsx:21]: why do we need this line? props?: any;

File: src/components/Tags/index.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+export interface ITagsState {
+  selectedTags: number[];
+}
+class Tags extends React.Component<ITagsProps, ITagsState> {
+  static defaultProps: any;
+  state = {
+    selectedTags: [],
+  };
+  handleChange(tagId: number) {
+    if (this.state.selectedTags.includes(tagId)) {
```
Comment (zaporozhetsAnton) [inline src/components/Tags/index.tsx:33]: what does this.state.selectedTags need for?

Comment (yuliiashpak) [reply src/components/Tags/index.tsx:33]: to save array of current selected tags ids. Well I forgot to add method onChange to return these values to parent component

Comment (yuliiashpak) [reply src/components/Tags/index.tsx:33]: method onChange added in last commit

File: src/components/Tags/Tag/index.tsx (hunk 1 above)
Comment (yuliiashpak) [reply src/components/Tags/Tag/index.tsx]: mistakenly added

**Pull request merged**